### DISCUSSION TITLE
We have an OS!

### DISCUSSION THREAD

**Discussion by Mauric3Moss** [S-PScripts/chromebook-utilities#22] [Ideas]
Hey @S-PScripts ,
First of all, thank you for making this list, this is awesome.
Second of all, A friend of mine made an OS, specifically for school Chromebook bypassing. I don't know if you want to include it, but we would be grateful if you did, as we are trying to gain more publicity. It is called VortexOS. Let us know, and thank you for your consideration.
[VortexOS](vortexos.net)
[VotexOS Unblocked](VortexOS.onrender.com)

-Kvothe, On Behalf of the VotexOS Dev Team

**Comment by S-PScripts** [comment]
I've added it now. You may view it here: https://github.com/S-PScripts/chromebook-utilities/blob/main/OS/VortexOS
